mason/site/footer/body.mas

```diff
@@ -1,20 +1,32 @@
+<%init>
+  my $tag = `git describe --abbrev=0 --tags`;
```

Comment:
this is a great idea!

in the site specific masons it is possible to do something like (this would also work in this repo):
my $dir = $c->config->{basepath};
my $tag = `cd $dir; git describe --abbrev=0 --tags`;
my $commit = `cd $dir; git rev-parse --short HEAD`;
my $updated = `cd $dir; git log -1 --format=%cd`;